apps/client/src/components/NotFoundContent/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+
+import { NotFoundTitle, ArrowIcon } from 'client/assets';
+
+import * as S from './style';
+
+const NotFoundContent = () => {
+  const router = useRouter();
+  
+  const handleClick = () => {
+    router.back();
+  };
+
```

Comment:
```suggestion
const NotFoundContent = () => {
  const { back } = useRouter();
  
  const handleClick = () => {
      back();
  };

```

구조분해 할당으로 코드를 줄일 수 있겠네요.

Reply:
suggestion 적용 했습니다 감사합니다!